app/components/ui/images.tsx

```diff
@@ -27,3 +27,113 @@ export function Image({ ...props }: ImageProps) {
     </div>
   );
 }
+
+interface ImageDetails {
+  url: string;
+  altText: string;
+  title?: string;
+}
+
+const staticImage: { [key: string]: ImageDetails } = {
+  avatarAstronaut: {
+    url: 'https://res.cloudinary.com/dxctpvd8v/image/upload/v1716394728/SocialPlanIt/avatar-astronaut_100x100.webp',
```

Comment:
I think we could even remove the `web` file ending, and Cloudinary would just use the best possible format. Not a blocker, but maybe something to experiment with!